changelog.md

```diff
@@ -5,6 +5,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+- [Address CVE-2023-33201 Bouncy Castle vulnerability](https://github.com/ballerina-platform/ballerina-standard-library/issues/4776)
+
+
+## [2.3.1] - 2023-06-01
+
+- This version maintains the latest dependency versions.
```

Comment:
what do u mean by this?

Reply:
We have updated some dependecy version during that release